server/app/middlewares/websocket_backend.rb

```diff
@@ -237,23 +237,22 @@ def subscribe_to_rpc_channel
 
   # @param [Hash] msg
   def on_rpc_message(msg)
-    client = client_for_id(msg['id'])
-    if client
-      self.send_message(client[:ws], msg['message'])
-    end
+    EM.next_tick{
+      client = client_for_id(msg['id'])
+      if client
+        self.send_message(client[:ws], msg['message'])
```

Comment:
This now `EM.next_tick`'s twice... once in the `on_rpc_message`, and then again in `send_message`... but I suppose that doesn't hurt.